athina_logger/tracing/span.py

```diff
@@ -0,0 +1,313 @@
+import datetime
+from typing import Any, Dict, List, Optional, Union
+from .models import SpanCreateModel
+from .util import remove_none_values
+
+class Span:
+
+    def __init__(
+        self,
+        name: str,
+        start_time: Optional[datetime.datetime] = None,
+        end_time: Optional[datetime.datetime] = None,
+        span_type: str = "span",
+        status: Optional[str] = None,
+        attributes: Optional[dict] = None,
+        input: Optional[dict] = None,
+        output: Optional[dict] = None,
+        duration: Optional[int] = None,
+        version: Optional[str] = None,
+    ):
+        if start_time is None:
+            start_time = datetime.datetime.utcnow()
+        else:
+            start_time = start_time.utcnow()
+        self._span = SpanCreateModel(
```

Comment:
Shall we rename this SpanCreateModel name? Just SpanModel and TraceModel would be better IMO. What are your thoughts on this @vivek-athina ?

Reply:
Just felt that the more explicit the name, lesser the confusion while using it